buildSrc/src/main/java/Dependencies.kt

```diff
@@ -2,13 +2,26 @@ const val android_min_sdk_version = 16
 const val android_target_sdk_version = 31
 
 const val kotlin_version = "1.6.21"
+const val agp_version = "7.1.3"
+const val ktlint_version = "10.2.0"
 
 const val ktor_version = "1.6.8"
```

Comment:
If we are going to put all the versions in here we might want to organize the file a bit.

Maybe it is just enough to have a separation with a comment line. Something like this (could be any other format):
```
# KMM

# Gradle plugins

# Android

# Testing
```

Reply:
Separated them